### PR TITLE
[Hub Generated] Review request for Microsoft.ManagedServices to add version preview/2020-02-01-preview

### PR DESCRIPTION
**Pull request by skayani**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/branch/skayani/azure-rest-api-specs/managedServicesJITSwaggerPublic...master/).

### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by skayani** [comment]
Received approval for exact same changes in private repo (https://github.com/Azure/azure-rest-api-specs-pr/pull/1192)

**Review by KrisBash — APPROVED** [review]
LGTM, thanks

File: specification/managedservices/resource-manager/readme.md [hunk 1]
```diff
@@ -26,18 +26,27 @@ These are the global settings for the ManagedServices API.
...
-tag: package-2019-09
+tag: package-preview-2020-02
 ```
 
 
+### Tag: package-preview-2020-02
+
+These settings apply only when `--tag=package-preview-2020-02` is specified on the command line.
+
+```yaml $(tag) == 'package-preview-2020-02'
```
Comment (ChenTanyi) [inline specification/managedservices/resource-manager/readme.md:37]: It seems you have format the yaml for package-2019-09, but the new yaml remains no space. Meanwhile, please change it into 2020-02-preview, thanks.

Comment (skayani) [reply specification/managedservices/resource-manager/readme.md]: Sorry this comment did not become resolved sooner.  We also had to wait to merge these changes due to code freeze + other delays.  Please let me know if anything else needs to be resolved.

File: specification/managedservices/resource-manager/readme.md [hunk 2]
```diff
@@ -26,18 +26,27 @@ These are the global settings for the ManagedServices API.
 
 ``` yaml
 openapi-type: arm
-tag: package-2019-09
+tag: package-preview-2020-02
```
Comment (ChenTanyi) [inline specification/managedservices/resource-manager/readme.md:29]: You need to change this tag too. The SDK generation cannot find this specific tag.

**Review by ChenTanyi — APPROVED** [review]

**Pull request merged**